packages/round-manager/src/assets/landing/banner.svg

```diff
@@ -1,4 +1,4 @@
-<svg width="704" height="800" viewBox="0 0 704 800" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+<svg width="704" height="700" viewBox="0 0 704 800" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
why was the height changed here?

Reply:
The original image I had exported didn't have the footer while the newer one did! 
That's why it's been updated